scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -1604,17 +1605,15 @@ public ExecutableContainer loadExecutableContainer(final InternalTask task) {
         return executeReadOnlyTransaction(session -> {
             Query query = session.getNamedQuery("findUsersWithJobs");
 
-            List list = query.list();
-            List<SchedulerUserInfo> users = new ArrayList<>(list.size());
-            for (Object obj : list) {
-                Object[] nameAndCount = (Object[]) obj;
-                users.add(new SchedulerUserInfo(null,
-                                                nameAndCount[0].toString(),
-                                                0,
-                                                Long.parseLong(nameAndCount[2].toString()),
-                                                Integer.parseInt(nameAndCount[1].toString())));
-            }
-            return users;
+            List<Object[]> list = query.list();
+
+            return list.stream()
+                       .map(nameAndCount -> new SchedulerUserInfo(null,
```

Comment:
Maybe you could use :
Arrays.steam(query.list()).map(nameAndCount ->...

Reply:
I will keep my version, cause Arrays.stream need array not list